code/datums/armor.dm

```diff
@@ -1,5 +1,45 @@
 #define ARMORID "armor-[melee]-[bullet]-[laser]-[energy]-[bomb]-[bio]-[fire]-[acid]-[wound]-[consume]"
 
+#define MAXIMUM_DAMAGE_RESIST 85 // 85%, set this to 100 to disable maximum resistance with DR but not recommended
+#define DAMAGE_THRESHOLD_MINIMUM 0 // 20%, set this to 0 to disable minimum damage w/ DT but not recommended
```

Comment:
This intentional, chief? You said yourself it's not reccomended.

Reply:
as mentioned in the description, Ninja wanted to test with 0 DT minimum so we're giving it a go